src/ResourceManagement/Automation/AutomationManagement/Generated/DscNodeConfigurationOperations.cs

```diff
@@ -236,7 +236,7 @@ public AutomationManagementClient Client
                     configurationValue["name"] = parameters.Configuration.Name;
                 }
                 
```

Comment:
Let me check this. My assumption was that this change was pushed to both Azure and AA forks.